moto/events/responses.py

```diff
@@ -409,3 +409,48 @@ def cancel_replay(self):
         result = self.events_backend.cancel_replay(name)
 
         return json.dumps(result), self.response_headers
+
+    def create_connection(self):
+        name = self._get_param("Name")
+        description = self._get_param("Description")
+        authorization_type = self._get_param("AuthorizationType")
+        auth_parameters = self._get_param("AuthParameters")
+
+        if authorization_type != "API_KEY":
+            raise NotImplementedError(
+                "EventsBridge with authorization_type {0} is not yet implemented".format(
+                    authorization_type
+                )
+            )
+
+        result = self.events_backend.create_connection(
+            name, description, authorization_type, auth_parameters
+        )
+
+        return (
+            json.dumps(
+                {
+                    "ConnectionArn": result.arn,
+                    "ConnectionState": "CREATING",
+                    "CreationTime": result.creation_time,
+                    "LastModifiedTime": result.creation_time,
+                }
+            ),
+            self.response_headers,
+        )
+
+    def list_connections(self):
+        connections = self.events_backend.list_connections()
+        result = []
+        for connection in connections:
+            result.append(
+                {
+                    "ConnectionArn": connection.arn,
+                    "ConnectionState": "CREATING",
```

Comment:
What's the end-state for connections? Looking at the docs, I'm assuming the happy flow is: CREATING -> AUTHORIZING -> AUTHORIZED, so we should probably return 'AUTHORIZED' here